src/Rand.php

```diff
@@ -76,8 +74,7 @@ public static function getAlternativeGenerator()
         if (!class_exists('RandomLib\\Factory')) {
             throw new Exception\RuntimeException(
                 'The RandomLib fallback pseudorandom number generator (PRNG) '
-                . ' must be installed in the absence of the OpenSSL and '
-                . 'Mcrypt extensions'
+                . ' must be installed in the absence of a secure source'
```

Comment:
This is a bit cryptic; as an end-user, if I don't know cryptography, I might not understand what "absence of a secure source" refers to. Is it cryptic on purpose? i.e., to prevent an attacker from understanding the capabilities of the server if the exception is displayed?